hardhat/SymfoniContext.tsx

```diff
@@ -0,0 +1,521 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
just a note that this isn't a required dependency. Ideally, we want to remove this context and cut it down prolifically. A lot of things in here that we do not need